### PR TITLE
:zap: added confirm password field

### LINKED ISSUE #116: Verify the new password
On the settings page for the user, it should be functionality to verify that the new password is equal. This means having two new password fields to compare each other.

### PR DESCRIPTION
**Pull request by PranjalAgni**
This PR adds the confirm password field and checks if the new password and confirm password is equal then only update the field.

This closes #116

### CODE REVIEW

File: src/server/controllers/account.js [hunk 1]
```diff
@@ -29,7 +29,13 @@ async function account(fastify) {
             preValidation: [fastify.authenticate],
         },
         async (request, reply) => {
-            const { currentPassword = '', password = '', email = '' } = request.body;
+            console.log(JSON.stringify(request.body, undefined, 3));
```
Comment (bjarneo) [inline src/server/controllers/account.js:32]: Please remove the console.log :)

**Review by bjarneo — CHANGES_REQUESTED** [review]
Looks good to me, only have to remove the console.log. 🚀

File: src/server/controllers/account.js (hunk 1 above)
Comment (PranjalAgni) [reply src/server/controllers/account.js:32]: Sure will remove it

Comment (PranjalAgni) [reply src/server/controllers/account.js]: Done

**Review by bjarneo — APPROVED** [review]

**Pull request merged**